Makefile

```diff
@@ -119,3 +121,6 @@ source/fixtures/swagger.yaml/%.refract.parse-result.1.0.yaml: source/fixtures/sw
 
 node_modules:
 	npm install --no-optional hercule dredd js-yaml media-typer chai fury-cli
+
+bundler:
```

Comment:
This should be PHONY because we don't have a file called bundler.

Reply:
Would be better if this target didn't affect global state and instead did a local install in the same directory. Then this wouldn't need to be PHONY.

Could set GEM_HOME to be inside this directory to achieve this.